src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_validators.py

```diff
@@ -943,41 +939,40 @@ def _get_default_address_pool(cli_ctx, resource_group, balancer_name, balancer_t
     return values[0]
 
 
-def _validate_vmss_create_load_balancer_or_app_gateway(cmd, namespace):
-    from msrestazure.azure_exceptions import CloudError
-    from msrestazure.tools import parse_resource_id
-    INSTANCE_THRESHOLD = _get_vmss_create_instance_threshold()
+def _validate_vmss_single_placement_group(namespace):
+    if namespace.platform_fault_domain_count is not None and namespace.zones is None:
+        raise CLIError('usage error: --platform-fault-domain-count <COUNT> --zones <ZONES>')
+    if namespace.platform_fault_domain_count == 1 or namespace.zones or namespace.instance_count > 100:
```

Comment:
Why did you remove the method that retrieves this value?  There was a test for this scenario that needed to be able to mock the threshold to something lower (since I believe creating the 100+ instance scale set actually just couldn't be done on our subscription due to quota limits).

Reply:
Can I leverage the `--validate` flag and check out the template?

Reply:
That might work, though I think it outputs the template as log.info so it might be tricky to get to.